webapp/config.py

```diff
@@ -24,3 +24,4 @@
 }
 
 WEBAPP = os.getenv('WEBAPP', 'snapcraft')
+WEBAPP_EXTRA_HEADER = {}
```

Comment:
Can this be made plural please? As the content is not a single item. `WEBAPP_EXTRA_HEADERS`